modules/logging-create-loki-cr-cli.adoc

```diff
@@ -54,7 +54,7 @@ It is not possible to change the number `1x` for the deployment size.
 // end::pre-5.9[]
 
```

Comment:
@xueli181114 If we have a section on Loki Operator deployment for OSD, I suggest checking the below docs on Openshift since there are changes to how Loki will be deployed on STS clusters from Logging 5.9 onwards
https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/installing-log-storage.html#installing-log-storage-loki-sts

Reply:
@kabirbhartiRH Do you mind to provide more info? Do you mean the doc content you pasted should be introduced?

Reply:
@xueli181114 On our OCM profiles job, I do not see an OSD template which has STS turned ON. Could you confirm if STS is available on OSD clusters running on AWS/Azure?

Reply:
@kabirbhartiRH STS only works for AWS cluster. Yes we can support STS for OSD cluster, but STS mainly used by rosa.

Reply:
@xueli181114 In case we support STS on OSD, I think it's better to include deploying Loki in STS mode.

Deploying the Operator in STS/Workload Identity Federation mode on AWS and Azure: https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/installing-log-storage.html#installing-log-storage-loki-sts

Section to create Loki Storage secret needed for LokiStack:

AWS: https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/installing-log-storage.html#AWS_storage_STS_installing-log-storage

Azure: https://docs.openshift.com/container-platform/4.15/observability/logging/log_storage/installing-log-storage.html#Azure_storage_STS_installing-log-storage

Reply:
Thank you @kabirbhartiRH . STS is supported by OSD but it is not showing in the official [doc](https://docs.openshift.com/dedicated/osd_install_access_delete_cluster/creating-an-aws-cluster.html). I would suggest another jira card to track STS for OSD support since it's a big change. And customers cannot create an sts OSD via UI.

Reply:
@eohartman Please note above comment :)

Reply:
@kabirbhartiRH Got it! I'll work on seeing if we already have a Jira ticket related to this, and if not, create one. Thank you!

Reply:
Marking as resolved, as this documentation gap will be covered under [OSDOCS-10494](https://issues.redhat.com/browse/OSDOCS-10494)

Reply:
@eohartman It seems this [section](https://docs.openshift.com/dedicated/observability/logging/log_storage/installing-log-storage.html#logging-identity-federation_installing-log-storage) is already present under OSD docs?